packages/zeebe-element-templates-json-schema/resources/schema.json

```diff
@@ -649,6 +649,31 @@
         "properties": {
           "$ref": "#/definitions/properties",
           "$id": "#/properties"
+        },
+        "icon": {
+          "$id": "#/icon",
+          "type": "object",
+          "description": "Custom icon to be shown on the element",
+          "default": {},
+          "properties": {
+            "contents": {
+              "$id": "#/icon/contents",
+              "type": "string",
+              "description": "The URL of an icon.",
+              "pattern": "^((https|http)://|data:).*",
```

Comment:
```suggestion
              "pattern": "^(https?|data):.*",
```

Reply:
Will never get used to regex... It can be so easy 😄